components/text/gettext/Makefile

```diff
@@ -42,6 +43,11 @@ COMPONENT_LICENSE_FILE=	COPYING
 
 include $(WS_MAKE_RULES)/common.mk
 
+PATH= $(PATH.gnu)
```

Comment:
Could you please add a comment explaining why is this needed?  Thank you.

Reply:
I thought that it was obvious that this change made two mode tests pass instead of skip.

Reply:
No, it is not.  Thanks for the comment.   However ... (to be continued)